modules/ossm-rn-new-features.adoc

```diff
@@ -19,11 +19,101 @@ Module included in the following assemblies:
 * *Policy Enforcement* - Apply organizational policy to the interaction between services, ensure access policies are enforced and resources are fairly distributed among consumers. Policy changes are made by configuring the mesh, not by changing application code.
 * *Telemetry* - Gain understanding of the dependencies between services and the nature and flow of traffic between them, providing the ability to quickly identify issues.
 
+== New features {SMProductName} 2.2
+
+This release of {SMProductName} adds new features and enhancements, and is supported on OpenShift Container Platform 4.9, and 4.10.
+
+=== Component versions included in {SMProductName} version {SMProductVersion}
+
+|===
+|Component |Version
+
+|Istio
+|1.12.7
+
+|Envoy Proxy
+|1.20.4
+
+|Jaeger
+|1.30.2
+
+|Kiali
+|1.48.0
```

Comment:
Kiali Contanier version is " v1.48.0-16".

Please update the Kiali version from " v1.48.0" to  "v1.48.0-16".

Referance: https://docs.google.com/document/d/16pspXhC6aJXtdn7FPJflKT-WuyhCMSkYwUmlRkqrBLI/edit#heading=h.gimd7qghbat8

Reply:
Thanks, @pbajjuri20, I've made the edit.